manifests/cli/reload.pp

```diff
@@ -8,13 +8,15 @@
     fail("Use of private class ${name} by ${caller_module_name}")
   }
 
+  $jar_file = $jenkins::cli::jar
```

Comment:
Why is this separate variable needed? It's only used in 1 place so I'd prefer it to be in the `require`.